src/v/cluster/health_monitor_frontend.cc

```diff
@@ -63,14 +64,14 @@ storage::disk_space_alert health_monitor_frontend::get_cluster_disk_health() {
     return _cluster_disk_health;
 }
 
-// Collcts and returns current node health report according to provided
-// filters list
+/**
+ * Gets cached or collects a node health report.
+ */
 ss::future<result<node_health_report>>
-health_monitor_frontend::collect_node_health(node_report_filter f) {
-    return dispatch_to_backend(
-      [f = std::move(f)](health_monitor_backend& be) mutable {
-          return be.collect_current_node_health(std::move(f));
-      });
+health_monitor_frontend::collect_node_health() {
```

Comment:
IMO the frontend method should also be called `get_current_node_health` - this makes it clear which node this refers to (in contrast to the RPC method, where it is more obvious - the node we are sending the request to).